isis/src/control/apps/jigsaw/jigsaw.xml

```diff
@@ -1513,5 +1513,106 @@
+               twist=yes camsolve=accelerations overexisting=yes camera_angles_sigma=.25 camera_angular_velocity_sigma=.1 camera_angular_acceleration_sigma=.01
+        </commandLine>
+        <description>
+        </description>
+      </terminalInterface>
+    </example>
+
+    <example>
+      <brief>Run of jigsaw parameterized for Kaguya Terrain Camera (TC) images and <b>an intermediate ground control</b> network covering the Apollo 15 landing site. </brief>
+      <description>
+        <p>
+         This example jigsaw bundle is run directly after adding ground control points to the previous relative network. With ground control points inserted into the 
+         bundle solution, we can expand the bundle solve parameters to attempt solving for point radius values and the position of the spacecraft.
+        </p>
+        <p>
+         Ground points are weighted more heavily in the bundle than relative control points and adding parameters adds more complexity to the bundle solution. Therefore, it 
+         is common to create and smith a network with only relative points and add ground point in AFTER the relative network (and its bundle solution) is of sufficient quality. 
+         The purpose of this bundle is to ensure the network bundle converges with the added ground points and solve parameters, before committing to updating the camera 
+         pointing on the images.
+        </p>
+        <p>
+         In addition to the previous solve parameters, this bundle turns on the point radius and spacecraft position solve parameters. Radius and spacecraft position 
+         are two parameters which can only be solved for when ground points are present in the bundle solution since they are concerned with absolute positioning.
+         Applying the point radius solve parameter requires the point_radius_sigma to be set. This value is a representation the uncertainty (in meters) of the 
+         cameras pointing corresponding to the correct elevation on the shape model. Often this value can be set, and the appropriateness of the set value can be 
+         checked using the 'POINTS DETAIL' section of the bundleout.txt file output by jigsaw. If the radius total correction of points exceeds the provided sigma, 
+         the uncertainty may need to be increased.
+        </p>
+        <p>
+         Applying the space craft position solve parameter requires an uncertainty estimation through spacecraft_position_sigma, while [file_prefix]_bundleout_images.csv 
+         holds the resulting image adjustments in the X Correction, Y Correction, and Z Correction columns. The overhermite flag allows a polynomial to estimate the 
+         spacecraft position, like the overexisting flag estimates the camera pointing with a polynomial. As such, this option saves memory and time at the expense of 
+         some loss in accuracy and this loss can be mitigated by switching it off for the final run.
```

Comment:
See my comments in the previous example. using `overexisting` or `overhermite` actually takes more memory than just solving for a polynomial as we have to store the a-priori ephemerides and the polynomial.